app/frontend/app/controllers/story.js

```diff
@@ -21,6 +21,14 @@ export default Ember.ObjectController.extend(HasCurrentUser, {
   loadingAll: false,
   loadedAll: propertyEqual('substories.length', 'model.substoryCount'),
 
+  // Determine if we are on the stories group page
+  // this is really hacky
+  isOnGroupPage: function() {
+    // 8 == /groups/
+    return this.get('model.group') &&
+      (this.get('target.target.url').substr(8) === this.get('model.group.id'));
```

Comment:
Do we have to do such a complex thing, can't we just check if the url just has "groups" in it?  I mean, it's not like posts to "Hello Kitty Fanclub" will be on pages for "Nuck's Yaoi Harem", right?